pkgs/servers/seafile/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub, pkg-config, curl, vala, python3, autoreconfHook
+, libuuid, sqlite, glib, libsearpc, libevent }:
+
+stdenv.mkDerivation rec {
+  pname = "seafile";
+  version = "7.0.9";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seafile";
+    rev = "v${version}";
+    sha256 = "1n0jq6d6vgk58qmqgdr7w7jfgcrlicnaafz1za9qf76sbi5vc6fk";
+  };
+
+  nativeBuildInputs = [ autoreconfHook python3 python3.pkgs.wrapPython ];
+
+  buildInputs = [ vala libuuid sqlite pkg-config glib libsearpc libevent curl ];
+
+  # Needed by seaf-cli
+  propagatedBuildInputs = with python3.pkgs; [ pysearpc future ];
+
+  postFixup = ''
+    wrapPythonPrograms
+  '';
+
+  meta = {
+    description =
+      "File syncing and sharing software with file encryption and group sharing, emphasis on reliability and high performance";
+    homepage = "https://github.com/haiwen/seafile";
+    license = lib.licenses.gpl2;
+    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [ greizgh ];
```

```suggestion
  meta = {
    description = "File syncing and sharing software with file encryption and group sharing, emphasis on reliability and high performance";
    homepage = "https://github.com/haiwen/seafile";
    license = licenses.gpl2;
    platforms = platforms.linux;
    maintainers = with maintainers; [ greizgh ];
```

gpl2Only or gpl2Plus?